test-configs.yaml

```diff
@@ -120,6 +120,12 @@ test_plans:
   v4l2:
```

Comment:
The commit subject "test-configs: run v4l2 with the stretch-v4l2 rootfs" should be changed to show it's for QEMU, as for other devices the v4l2 test plan is already run on stretch-v4l2.